### ISSUE TITLE
Curl is incorrect when using relative spec-url

### ISSUE THREAD

**Issue by croudet** [mrin9/RapiDoc#15]
When using relative `spec-url` the curl is missing the server part:
```
<rapi-doc id="rapidoc" show-header="false" theme="dark" layout="row" spec-url="/swagger/spec-1.0.yml"></rapi-doc>
```
```
curl -X GET "/ds/assets?"
```

**Comment by mrin9** [comment]
can you confirm if `servers` or `host` is specified in the spec ?

**Comment by croudet** [comment]
There is no `servers` or `host` in the spec file.

**Comment by mrin9** [comment]
Is your, 'TRY' API call working ?

However, If it is missing in the spec, and you dont have any control on the spec, you can define the server in the `<rapi-doc>` element using `server` attribute

Let me know, if this is not the solution you like to settle for and have any alternate suggestion

**Comment by croudet** [comment]
The `Try API` calls are working.
The same spec file with swagger-ui gives : `curl -X GET "http://<server>:<port>/ds/assets" -H  "accept: */*"`

**Comment by mrin9** [comment]
Can you help find out the url generated by `swagger-ui` is relative to where you are hosting the `.html` or the API spec

Say you your html is at `http://example.com/api-doc.html` and your spec is at  `http://example.com/spec/petstore.json` .

`<rapi-docs>` The Try API uses the `.html's` location

**Comment by croudet** [comment]
The spec is hosted on the same server as the html. It is specified by a relative url:
`<rapi-doc id="rapidoc" show-header="false" theme="dark" layout="row" spec-url="/swagger/diff-server-1.0.yml"/>`

**Comment by mrin9** [comment]
released 4.2.2, which should take care of this issue. Feel free to re-open in case it isnt working the way you expect it to be
